tensornetwork/linalg/initialization.py

```diff
@@ -17,6 +17,8 @@
 from typing import Optional, Sequence, Tuple, Any, Union, Type, Callable, List
 from typing import Text
 import numpy as np
+from tensorflow.python.ops.numpy_ops.np_arrays import convert_to_tensor
```

Comment:
I know we haven't completely followed this rule, but ideally one should only import modules and not functions directly